app/service/rest.py

```diff
@@ -344,6 +345,45 @@ def get_all_notifications_for_service(service_id):
     ), 200
 
 
+@service_blueprint.route('/<uuid:service_id>/notifications/csv', methods=['GET'])
+def get_all_notifications_for_service_csv(service_id):
```

Comment:
Obviously this needs to be refactored so that all the code is reused rather than copied.